src/materialized/lib.rs

```diff
@@ -24,14 +24,14 @@ static ALLOC: jemallocator::Jemalloc = jemallocator::Jemalloc;
 use std::any::Any;
 use std::net::{IpAddr, Ipv4Addr, Ipv6Addr, SocketAddr};
 use std::path::PathBuf;
-use std::sync::Arc;
 use std::thread;
 use std::time::{Duration, Instant};
 
 use compile_time_run::run_command_str;
 use failure::format_err;
 use futures::channel::mpsc;
 use openssl::ssl::{SslAcceptor, SslFiletype, SslMethod};
+use stream_cancel::{StreamExt as _, Trigger, Tripwire};
```

Comment:
we don't normally voldemortize imported traits even if we're just using them for methods, I don't think, did this cause a problem or did it just feel right? Doesn't need to be changed, just curious.

Reply:
Oh, good catch. When I wrote this code `futures::stream::StreamExt` was also in scope, but that got refactored away.